config/base.js

```diff
@@ -59,6 +60,6 @@ module.exports = {
     'space-infix-ops': 'error',
     'template-curly-spacing': 'error',
 
-    'no-only-tests/no-only-tests': 2
```

Comment:
Pretty sure Chad originally added this for tape tests or something, so check with him on removing it -- might need to keep it, or move it to the node config, or maybe there's another linting library that would work better for his use cases?

Reply:
It looks like neither the old plugin or the new one are sophisticated enough to catch both use cases (QUnit and mocha/node) and we might as well leave everything in the base config

Reply:
👍